tests/test_sessions.py

```diff
@@ -50,6 +50,48 @@ def test_aggregates(sentry_init, capture_envelopes):
     )
     envelopes = capture_envelopes()
 
+    with sentry_sdk.isolation_scope() as scope:
+        with track_session(scope, session_mode="request"):
+            try:
+                scope.set_user({"id": "42"})
+                raise Exception("all is wrong")
+            except Exception:
+                sentry_sdk.capture_exception()
+
+    with sentry_sdk.isolation_scope() as scope:
+        with track_session(scope, session_mode="request"):
+            pass
+
+    sentry_sdk.get_isolation_scope().start_session(session_mode="request")
+    sentry_sdk.get_isolation_scope().end_session()
+    sentry_sdk.flush()
```

Comment:
Tests are always also a kind of documentation on how to use the code. 

Could you add comments to the three sessions to mark what aggregate status is expected for each one of them?

Please add those comments to all similar tests in this file. This will help us in the future (or others) to understand what is going on.